packages/scaffolding-cli/templates/src/ssr/compositions/home/index.tsx

```diff
@@ -27,9 +27,6 @@ interface Props
         ReturnType<typeof mapDispatchToProps> {}
 
 const Home: FC<Props> = ({isLoading, menuItems, getMenulist}) => {
-    useEffect(() => {
-        getMenulist({searchTerm: ""})
-    }, [])
```

Comment:
Would be cool to have a test to show that this call is only happening once now.

Reply:
Sorry this was a old implementation. I forgot to remove after the auto search implementation. So this is redundant. Did that answer ?